WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/wooshippinglabels/split/WooShippingSplitShipmentScreen.kt

```diff
@@ -487,6 +489,31 @@ fun SelectableProductsSection(
     modifier: Modifier = Modifier
 ) {
     Column(modifier = modifier) {
+        if (shipment.purchased) {
+            Column(
+                modifier = Modifier
+                    .padding(bottom = 16.dp)
+                    .fillMaxWidth()
+                    .background(
+                        color = colorResource(R.color.woo_shipping_label_success_surface),
+                        shape = RoundedCornerShape(8.dp),
+                    )
+                    .padding(16.dp),
+                verticalArrangement = Arrangement.spacedBy(4.dp)
+            ) {
+                Text(
+                    text = stringResource(R.string.woo_shipping_split_shipment_purchased_message_title),
+                    color = colorResource(R.color.woo_shipping_label_success),
+                    style = MaterialTheme.typography.body2.copy(fontWeight = FontWeight.SemiBold)
```

Comment:
np, `Text` has a `fontWeight` argument:


```suggestion
                    style = MaterialTheme.typography.body2,
                    fontWeight = FontWeight.SemiBold
```

Reply:
Applied! 👍🏻